comp/otelcol/converter/impl/testdata/receivers/empty-staticconfigs/config-result.yaml

```diff
@@ -0,0 +1,40 @@
+receivers:
+    prometheus/user-defined:
+      config:
```

Comment:
Your YAML files all have 4 spaces instead of 2 for the first indentation level, I assume this isn't intentional?